platforms/core-runtime/instrumentation-reporting/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
```

Comment:
💭 This project contains the main code for reporting, then it's just wired in the right places